routes/course_routes.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var handleError = require(__dirname + '/../lib/handle_server_error.js');
+var jsonParser = require('body-parser').json;
+var Course = require(__dirname + '/../models/course.js');
+
+
+var courseRouter = module.exports = exports = express.Router();
+
+courseRouter.get('/all-courses', function(req, res) {
```

Comment:
same thing here, unless there's already a get to courses, there's no need for an all-courses.